### PR TITLE
Replaced c3 commands with partial

### PR DESCRIPTION
**Pull request by kodster28**
### Summary

As part of #15825 (and I'm sure another c3 PR), we updated the flow for C3.

This PR does the following:

- [x] Moves more C3 commands to a partial that also includes the directory of the newly created application (one less step in C3, so less cognitive overhead).
- [x] Adds relevant `cd` commands, which is possible b/c we're specifying the directory more often + also reduces the cognitive overhead.
- [x] Restructures C3 prompt guidance into a partial that can more easily be updated / maintained in the future.

### What it doesn't do

For now, avoiding the edge cases of more arguments in C3 commands + Framework guides step-by-step instructions.

Those are sparingly used, can address in the future.

### CODE REVIEW

File: content/cloudflare-one/identity/authorization-cookie/cors.md [hunk 1]
```diff
@@ -139,72 +139,71 @@ Follow [these instructions](/cloudflare-one/identity/service-tokens/) to generat
...
-   ```
+This will prompt you to install the [`create-cloudflare`](https://www.npmjs.com/package/create-cloudflare) package and lead you through setup.
 
-4. Deploy the Worker to your Cloudflare account:
+{{<render file="_c3-post-run-steps.md" productFolder="workers" withParameters="Hello World example;;Hello World Worker;;JavaScript">}}
 
-   ```sh
-   $ npx wrangler@latest deploy
-   ```
+Go to your project directory.
```
Comment (kodster28) [inline content/cloudflare-one/identity/authorization-cookie/cors.md:150]: Github is weird about this piece... it's just adding this part before, don't know why it shifted down so much.

File: content/workers/tutorials/handle-form-submissions-with-airtable/index.md [hunk 2]
```diff
@@ -84,18 +84,9 @@ To handle the form submission, create and deploy a Worker that parses the incomi
...
-$ npm create cloudflare@latest airtable-form-handler
-```
-
-When configuring your Worker:
+{{<render file="_c3-run-command-with-directory.md" productFolder="workers" withParameters="airtable-form-handler">}}
 
-* Choose `"Hello World" Worker` for the type of application you would like to create.
-* Select `No` to using TypeScript.
-* Select `No` to deploying your Worker.
+{{<render file="_c3-post-run-steps.md" productFolder="workers" withParameters="Hello World example;;Hello World Worker;;JavaScript">}}
```
Comment (pedrosousa) [inline content/workers/tutorials/handle-form-submissions-with-airtable/index.md:90]: Should we also `cd` into the project directory here?

File: content/workers/tutorials/store-data-with-fauna/index.md [hunk 3]
```diff
@@ -103,38 +103,21 @@ Server keys can read and write all documents in all collections and can call all
...
-header: Create a new project
----
-$ npm create cloudflare@latest
-```
+To continue with this guide:
 
-{{</tab>}}
-{{<tab label="yarn" >}}
+- For *What would you like to start with*?, select `Framework Starter`.
+- For *Which development framework do you want to use?*, select `Hono`
```
Comment (pedrosousa) [inline content/workers/tutorials/store-data-with-fauna/index.md]:
```suggestion
- For *Which development framework do you want to use?*, select `Hono`.
```

File: archetypes/tutorial.md [hunk 4]
```diff
@@ -30,9 +30,7 @@ Before you start, make sure you have:
 
 First, use the `c3` CLI to create a new Cloudflare Workers project.
 
-```sh
-$ npm create cloudflare@latest <PROJECT_NAME>
-```
+{{<render file="_c3-run-command-with-directory" productFolder="workers" withParameters="<PROJECT_NAME>">}}
```
Comment (pedrosousa) [inline archetypes/tutorial.md]:
```suggestion
{{<render file="_c3-run-command-with-directory.md" productFolder="workers" withParameters="<PROJECT_NAME>">}}
```

File: content/pages/tutorials/localize-a-website/index.md [hunk 5]
```diff
@@ -32,17 +32,13 @@ If you would like to deploy your own version of the site, you can find the sourc
...
 
 For setup, select the following options:
 
-- Where do you want to create your application?: Input i18n-example.
-- What type of application do you want to create?: Select "Website or web app".
-- Which development framework do you want to use? Select "React"
-- Do you want to use TypeScript?: Select No.
-- Do you want to deploy your application?: Select No.
+- For *What would you like to start with*?, select `Framework Starter`.
+- For *Which development framework do you want to use?*, select `React`
```
Comment (pedrosousa) [inline content/pages/tutorials/localize-a-website/index.md]:
```suggestion
- For *Which development framework do you want to use?*, select `React`.
```

File: content/workers/tutorials/build-a-slackbot/index.md [hunk 6]
```diff
@@ -95,38 +95,22 @@ When your webhook is created, it will attempt to send a test payload to your app
...
-header: Create a new project
----
-$ npm create cloudflare@latest
-```
+Follow these steps to create a Hono project.
 
-{{</tab>}}
-{{<tab label="yarn" >}}
+- For *What would you like to start with*?, select `Framework Starter`.
+- For *Which development framework do you want to use?*, select `Hono`
```
Comment (pedrosousa) [inline content/workers/tutorials/build-a-slackbot/index.md]:
```suggestion
- For *Which development framework do you want to use?*, select `Hono`.
```

File: content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md [hunk 7]
```diff
@@ -37,7 +37,7 @@ Open a terminal window and run C3 to create your Worker project. This example cr
 
 
 ```sh
-$ npm create cloudflare@latest customer-worker-1 -- --type=hello-world
+$ npm create cloudflare customer-worker-1 -- --type=hello-world
```
Comment (pedrosousa) [inline content/cloudflare-for-platforms/workers-for-platforms/get-started/configuration.md]: Not sure when we need to have `@latest` or not. Here we're removing.

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**